Packs/CofenseIntelligenceV2/Integrations/CofenseIntelligenceV2/CofenseIntelligenceV2.py

```diff
@@ -0,0 +1,560 @@
+        threats = result.get('data', {}).get('threats', [])
+        remove_false_vendors_detections_from_threat(threats)
+        outputs = {'Data': ip, 'Threats': threats}
+        dbot_score_obj = Common.DBotScore(indicator=ip, indicator_type=DBotScoreType.IP,
+                                          integration_name=INTEGRATION_NAME, score=0,
+                                          reliability=params.get(RELIABILITY))
+        md_data, dbot_score, ip_indicator = ip_threats_analysis(threats=threats, ip=ip,
+                                                                threshold=params.get("ip_threshold"),
+                                                                dbot_score_obj=dbot_score_obj)
+
+        dbot_score_obj.score = dbot_score
+        ip_indicator.dbot_score = dbot_score_obj
+
+        command_results = CommandResults(
+            outputs_prefix=f'{OUTPUT_PREFIX}.IP',
+            outputs_key_field='Data',
+            outputs=outputs,
+            raw_response=result,
+            readable_output=tableToMarkdown(name=f'Cofense IP Reputation for IP {ip}', t=md_data,
+                                            headers=['Threat ID', 'Threat Types', 'Verdict', 'Executive Summary',
+                                                     'Campaign', 'Last Published', 'ASN', 'Country', 'Threat Report']),
+            indicator=ip_indicator, )
+        results_list.append(command_results)
+    return results_list
+
+
+def check_email_command(client: Client, args: Dict[str, Any], params) -> List[CommandResults]:
+    """ Performs the api call to cofense threts-search endpoint to get all threats associated with the given email,
+     analyze the response and generates the command result object for the email command
+            Args:
+                - client (Client): client instance that is responsible for connecting with cofense API
+                - args (Dict): the command args- email
+                - params (Dict): The integration params such as threshold, reliability
+            return:
+             CommandResults: results of the email command including outputs, raw response, readable output
+    """
+
+    emails = argToList(args.get('email'))
+    days_back = args.get('days_back') if args.get('days_back') else params.get('days_back')
+    if not emails:
+        raise ValueError('Email not specified')
+    results_list = []
+    for email in emails:
+        if not re.fullmatch(EMAIL_REGEX, email):
+            raise ValueError(f'Invalid email address: {email}')
+
+        # Call the Client function and get the raw response
+        result = client.threat_search_call(email=email, days_back=days_back)
+        threats = result.get('data', {}).get('threats', [])
+        remove_false_vendors_detections_from_threat(threats)
+        outputs = {f'{OUTPUT_PREFIX}.Email.Data': email, f'{OUTPUT_PREFIX}.Email.Threats': threats,
+                   'Account.Email.Address': email, 'Account.Domain': email.split('@')[1]}
```

Comment:
Please use the Account indicator type from CommonServerPython (same as you've done for Email here)